src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEntryPointSymbol.cs

```diff
@@ -334,14 +334,13 @@ private static BoundCall CreateParameterlessCall(CSharpSyntaxNode syntax, BoundE
         internal sealed class AsyncForwardEntryPoint : SynthesizedEntryPointSymbol
         {
             // The user-defined asyncrhonous main method.
-            private MethodSymbol _userMain;
+            private readonly MethodSymbol _userMain;
             private DiagnosticBag _diagnosticBag;
```

Comment:
Have we figured out my earlier questionts about this bag in general?  It seems like it's opening the door for duplicate / unnecessary errors here.  Feel like they should be resolved at the point of creating this symbol, not later.